src/include/DuckCrypto.h

```diff
@@ -20,19 +20,19 @@ namespace duckcrypto {
        * @brief Enable or disable encryption flag.
        *
        */
-      bool encryptOn = false;
+      [[maybe_unused]] bool encryptOn = false;
 
       /**
        * @brief Enable or disable decryption flag.
        *
        */
-      bool decryptOn = false;
+      [[maybe_unused]] bool decryptOn = false;
 
       /**
        * @brief Default encryption key, can be set in application layer using `void setAESKey(uint8_t newKEY[32])`.
        *
        */
-      uint8_t KEY[32] = {0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
+      [[maybe_unused]] uint8_t KEY[32] = {0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
```

Comment:
These are all actually used in DuckCrypto.cpp
For example

```
   void setEncrypt(bool state) {
      encryptOn = state;
   }

   bool getState() { return encryptOn; }

   void setDecrypt(bool state) {
      decryptOn = state;
   }
   ```
   
   The KEYS are also used there